### ISSUE TITLE
Documentation for branch PG16 is missing

### ISSUE THREAD

**Issue by michaelpq** [ossc-db/pg_hint_plan#152]
The documentation is available currently for HEAD, PG15 and PG14:
- https://pg-hint-plan.readthedocs.io/en/latest/
- https://pg-hint-plan.readthedocs.io/en/pg15/
- https://pg-hint-plan.readthedocs.io/en/pg14/
However it is missing for PG16. Could it be possible to add the missing webhooks to build on this branch as well.

Also, "latest" points to HEAD, but shouldn't it be PG16 instead? Publishing the docs of HEAD make sense to me, but this is not the latest version released, just the current version in development. @rjuju @yamatattsu , what do you think can be done about that?

**Comment by mikecaat** [comment]
Hi,

I tried to add PG16 and master branch for the docs.
* https://pg-hint-plan.readthedocs.io/en/master/
* https://pg-hint-plan.readthedocs.io/en/pg16/

And changed the "latest" points to PG16.
* https://pg-hint-plan.readthedocs.io/en/latest/

> Could it be possible to add the missing webhooks to build on this branch as well.

It wasn't necessary. I just had to change the settings of "Read the Docs" because it seems that webhooks only need to be set up per project and not per branch.

@rjuju I also add @SeinoYuki as a maintainer of "Read the Docs" projects because he will maintain the webhooks.
